Custom Axios options (request config) (#213)
SAP/cloud-sdk: docs/js/features/odata/execute-odata-request.mdx

```

This will add the custom service path to your request.

> Executing the example request above against a destination with URL `https://my.s4-system.com` would result in a request against URI like this: `https://my.s4-system.com/my/custom/service/path/MyEntity`.

Reviewer: ```suggestion
Executing the example request above against a destination with URL `https://my.s4-system.com` would result in a request against something like this: `https://my.s4-system.com/my/custom/service/path/MyEntity`.
```

> Executing the example request above against a destination with URL `https://my.s4-system.com` would result in a request against URI like this: `https://my.s4-system.com/my/custom/service/path/MyEntity`.

Reviewer: ```suggestion
Executing the example request above against a destination with URL `https://my.s4-system.com` would result in a request against a URL like this: `https://my.s4-system.com/my/custom/service/path/MyEntity`.
```

Reviewer: I left two suggestions, your choice ;) You can also adjust it differently, what is important to me is the missing "a".

### Setting a Custom Request Configuration
<CustomRequestConfig />

```ts
MyEntity.requestBuilder()
  .getAll()
  .addCustomRequestConfiguration({ responseType: 'arraybuffer' })
```

> <CustomRequestConfigNote />

Reviewer: As you have touched this file consider fixing the warnings below.

This will add the custom service path to your request.

> Executing the example request above against a destination with URL `https://my.s4-system.com` would then execute the request against something like this: `https://my.s4-system.com/my/custom/service/path/MyEntity`.

Reviewer 2: ```suggestion
Executing the example request above against a destination with URL `https://my.s4-system.com` would result in a request against URI like this: `https://my.s4-system.com/my/custom/service/path/MyEntity`.
```

### Setting a Custom Request Configuration
<CustomRequestConfig />

```ts

> MyEntity.requestBuilder()

Author: The example cannot be used as this is odata like. For openapi, we need a similar but different example.
